### DISCUSSION TITLE
Bug-Fix Only Version?

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1369] [Feature Requests]
@rern 
Would it be possible to have 6 months or so between any updates which include new features? The older versions of rAudio did seem to have fewer problems. Unavoidably, as the options and features increase, so do the bugs and regressions. I am very tempted to go back to an earlier, simpler, version, but many of the features I will never use have come with brilliant enhancements to the basic functions which I would not want to lose. The current versions have so many different user configurations and features that comprehensive testing before release is clearly not practical.

Having 6 months of updates which only fix bugs and regressions, alongside a beta version to test new features, would save a lot of frustration.

**Comment by Rasb74** [comment]
Totally agree, 
Usually in software dev once a version is validated as fully workable it is assign as "STABLE" version, the others are "BETA" or for developers only. 
I suggest an area to load the STABLE versions separately 
congrats for the job to RERN and still collaborative project !

**Comment by kw-sudo** [comment]
My r/audio requirements are also rudimentary (NAS-LAN/USB-amp, bitperfect, working library).

I've spent the last few days (some would say 'wasted') trying to heed all the previous - grateful - tips in various variations to install the current version without errors (keyword: library view). I am now stuck at i20230521 (without addon update), where it still works.

ceejayemm already talked about current "on/off errors". I would want to take work off rern also in that respect, not to have to develop the „egg-laying wooly pig“. Just the previous simplicity of r/Audio as a high-res media player I found appealing.
